package.json

```diff
@@ -84,24 +70,18 @@
     "invariant": "^2.2.0",
     "json-loader": "^0.5.4",
     "lint-staged": "^0.1.1",
-    "markdown-it": "^5.0.3",
-    "markdown-it-toc-and-anchor": "^1.0.1",
     "postcss-cssnext": "^2.4.0",
     "postcss-loader": "^0.7.0",
     "pre-commit": "^1.1.2",
-    "react": "^0.14.7",
-    "react-dom": "^0.14.7",
+    "react": "^15.0.0-rc.1",
+    "react-dom": "^15.0.0-rc.1",
```

Comment:
React master branch is constantly used in production at Facebook, so we cannot say that it's "not stable". :)